packages/@dcl/inspector/src/components/Resizable/types.ts

```diff
@@ -1,5 +1,34 @@
 export interface PropTypes {
-  minWidth?: 'initial' | number
-  initialWidth?: number
+  type: 'horizontal' | 'vertical'
+  min?: 'initial' | number
+  initial?: number
+  max?: number
   onChange?(value: [number, number]): void
 }
+
+export interface TypeProps {
+  offsetValue: 'offsetWidth' | 'offsetHeight'
+  eventClientValue: 'clientX' | 'clientY'
+  css: {
+    childs: 'width' | 'height'
+    handle: 'left' | 'top'
+  }
+}
+
+export const HORIZONTAL_PROPS: TypeProps = {
```

Comment:
these are not types 👀

Reply:
yes, but they're not important enough to live in their own file and I don't want them in the actual component either, so until this component grows bigger, I think this file is the best for now